public/app/features/dashboard/components/ShareModal/SharePublicDashboard.tsx

```diff
@@ -65,10 +65,7 @@ export const SharePublicDashboard = (props: Props) => {
     usage: false,
   });
 
-  const timeRange = getTimeRange(
-    { from: props.dashboard.getDefaultTime().from, to: props.dashboard.getDefaultTime().to },
-    props.dashboard.timezone
-  );
```

Comment:
this code was crashing for me as it was sending in props.dashboard.timezone but this function expects a TimeModel as second param (usually a dashboard model passed in). Think typescript allowed this due to bad any typing of dashboard.timezone prop